src/scatra_ele/4C_scatra_ele_calc_multiporo_reac.hpp

```diff
@@ -476,10 +476,10 @@ namespace Discret
         volfracpressure_ = phasemanager_->vol_frac_pressure();
 
         //! convective velocity
-        std::vector<Core::LinAlg::Matrix<nsd, 1>> phase_fluid_velocity(0.0);
+        std::vector<Core::LinAlg::Matrix<nsd, 1>> phase_fluid_velocity(true);
```

Comment:
Here your regex was wrong: While the existing case is of course super strange, please change this to:

```
std::vector<Core::LinAlg::Matrix<nsd, 1>> phase_fluid_velocity(numfluidphases + numvolfrac);
```

You are creating a vector of matrices here.